skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -10,40 +10,12 @@ import platform.UIKit.UIEvent
 import platform.UIKit.UITouch
 import platform.UIKit.UIScreen
 import platform.UIKit.UIView
-import platform.UIKit.UIViewController
 import platform.UIKit.setFrame
 import platform.UIKit.contentScaleFactor
 import platform.UIKit.UIKeyInputProtocol
 import platform.UIKit.UIPress
 import platform.UIKit.UIPressesEvent
 
-@ExportObjCClass
-class SkikoViewController : UIViewController {
```

Comment:
Why isn't it needed in library? Writing it every time on the user side looks like not the right thing.

Reply:
Yeah, we may stay it in library. I will bring it back.

Reply:
Done